### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by yalh76**
## Problem

- *Level0*

## Solution

- *Apply example_ynh*
- *Fix ImportError: cannot import name 'soft_unicode' from 'markupsafe'*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -3,34 +3,52 @@
...
+#=================================================
+
 domain=$YNH_APP_ARG_DOMAIN
 path_url=$YNH_APP_ARG_PATH
 admin=$YNH_APP_ARG_ADMIN
+admin_pwd=$YNH_APP_ARG_PASSWORD
+
 db_user="pgadmin"
-admin_pwd=$YNH_APP_ARG_ADMIN_PASSWORD
+db_pwd=$(ynh_string_random --length 30)
```
Comment (Josue-T) [inline scripts/install:33]: The user need this password, we can't use a random password.

File: manifest.json [hunk 2]
```diff
@@ -25,40 +30,21 @@
...
             },
             {
-                "name": "admin_password",
-                "type": "password",
-                "ask": {
-                    "en": "Enter a password for the administrator",
-                    "fr": "Entrez un mot de passe pour l'administrateur"
-                },
-                "example": "**Sup3rS3cr3t**"
+                "name": "password",
```
Comment (Josue-T) [inline manifest.json:46]: Why do you remove the password Here ?

We still need it !!!!

**Review by Josue-T — COMMENTED** [review]
Are you sure that the upgrade from all previous version are OK ?

File: scripts/install (hunk 1 above)
Comment (yalh76) [reply scripts/install:33]: it's still here, L30

File: manifest.json (hunk 2 above)
Comment (yalh76) [reply manifest.json:46]: Password is not removed, I just change the name of the variable from `admin_password` to `password`

**Comment by yalh76** [comment]
> Are you sure that the upgrade from all previous version are OK ?

not that sure, as previous versions no more install ...

File: manifest.json (hunk 2 above)
Comment (Josue-T) [reply manifest.json:46]: Ah yes didn't get it

**Comment by Josue-T** [comment]
> > Are you sure that the upgrade from all previous version are OK ?
> 
> not that sure, as previous versions no more install ...

So you merge a pull request in fiew days in master and you are not sure that the actual installation will be able to be upgraded ?

**Comment by yalh76** [comment]
> > > Are you sure that the upgrade from all previous version are OK ?
> > 
> > 
> > not that sure, as previous versions no more install ...
> 
> So you merge a pull request in fiew days in master and you are not sure that the actual installation will be able to be upgraded ?

the changes I made seems ok, but I can't be 100% sure because previous version no more install cause of  `ImportError: cannot import name 'soft_unicode' from 'markupsafe'`  https://ci-apps.yunohost.org/ci/job/6181

If you prefer I can revert the merge I made.

**Pull request merged**